src/html/index.html

```diff
@@ -83,6 +84,7 @@
                 <li class="active"><a href="#game" data-toggle="tab" role="tab" aria-controls="home" aria-selected="true">Game</a></li>
                 <div class="btn-group action-buttons">
                     <button id="fullscreen" type="button" class="btn btn-default fullscreenbutton">Full Screen</button>
+                    <button class="form-check"><input class="form-check-input" type="checkbox" value="" id="Grid" /> <label class="form-check-label" for="Grid">Grid</label></button>
```

Comment:
Cool. Looks good to me. This is a good spot. We are using bootstrap 3 (maybe we should update to bootstrap 5), but if you want to do anymore styling checkout the bootstrap docs: https://getbootstrap.com/docs/4.0/components/forms/#checkboxes-and-radios